scripts/linux/trcE2ETest.sh

```diff
@@ -291,18 +291,29 @@ function print_deployment_logs() {
     docker logs edgeAgent || true
 }
 
+
+function get_support_bundle_logs(){
+    
+    echo "Getting Support Bundle Logs"
+    mkdir -p $working_folder/support
+    iotedge support-bundle -o $working_folder/support/iotedge_support_bundle.zip
+    echo "finished getting support Bundle Logs"
+
+}
+
 function print_test_run_logs() {
     local ret=$1
 
     print_highlighted_message "test run exit code=$ret"
     print_highlighted_message 'Print logs'
     print_highlighted_message 'testResultCoordinator LOGS'
     docker logs testResultCoordinator || true
+    get_support_bundle_logs
```

Comment:
I think getting the support bundle can replace most of the existing implementation of `print_test_run_logs`--basically just print out the test run exit code, get the support bundle, and exit. We can do that work in a different PR, but getting the support bundle _and_ doing all these `docker logs` calls at the same time is really going to stress the agent, and it will take a looonng time.

Reply:
It should probably replace `print_deployment_logs` too (although I don't have much experience with these tests... @and-rewsmith would know for sure).

Reply:
Given that support bundle logs contain all the necessary logs, I will remove the print_test_run_logs.